cime_config/allactive/config_pesall.xml

```diff
@@ -270,6 +270,125 @@
       </pes>
     </mach>
   </grid> <!-- end grid name="any" -->
+  <!-- 2000_DATM%JRA_ELM%SPBC_MPASSI_MPASO%DATMFORCED_MOSART_SGLC_SWAV_SIAC_SESP -->
+  <!-- ATM_GRID is ne30np4.pg2  ICE_GRID is EC30to60E2r2 -->
+  <grid name="a%ne30np4">
+    <mach name="pm-cpu">
+      <pes compset="JRA_ELM.+MPASSI.+MPASO.+MOSART.+SGLC.+SWAV" pesize="any">
+        <comment>"pm-cpu 4 nodes, 256 partition"</comment>
+        <ntasks>
+          <ntasks_atm>-4</ntasks_atm>
+          <ntasks_lnd>-4</ntasks_lnd>
+          <ntasks_rof>-4</ntasks_rof>
+          <ntasks_ice>-4</ntasks_ice>
+          <ntasks_ocn>-4</ntasks_ocn>
+          <ntasks_glc>-1</ntasks_glc>
+          <ntasks_wav>-1</ntasks_wav>
+          <ntasks_cpl>-4</ntasks_cpl>
+        </ntasks>
+      </pes>
+    </mach>
+    <mach name="cori-haswell">
```

Comment:
Did you mean to bring in these layouts for other machines?

Reply:
I was trying to only make updates affecting pm-cpu (and pm-gpu/alvarez), but I actually wasn't sure if it make more sense to bring in "all" new changes into files describing PE layouts as it's easier to maintain.  But then there may be some change that is required elsewhere...

Reply:
Just mention it in the PR description.